test/controllers/trips_controller_test.rb

```diff
@@ -1,23 +1,137 @@
 require "test_helper"
 
 describe TripsController do
+  before do
+    @driver = Driver.create(name: "Driver 1", vin: "1234")
+    @passenger = Passenger.create(name: "Passenger 1", phone_num: "1234")
+
+    @trip = Trip.create(date: Date.today, passenger_id: @passenger.id, driver_id: @driver.id, rating: nil, cost: 1234)
+  end
+
   describe "show" do
-    # Your tests go here
+    it "responds with success when showing an existing valid trip" do
+      id = @trip.id
+
+      get trip_path(id)
+
+      must_respond_with :success
+    end
+
+    it "responds with 404 with an invalid trip id" do
+      get trip_path(-1)
+
+      must_respond_with :not_found
+    end
   end
 
   describe "create" do
-    # Your tests go here
+    it "can create a new trip" do
+      driver = Driver.create(name: "Driver 1", vin: "1234")
+      passenger = Passenger.create(name: "Passenger 1", phone_num: "1234")
+
+      new_trip = {
+          trip: {
+            date: Date.today,
+            passenger_id: passenger.id,
+            driver_id: driver.id,
+            rating: nil,
+      }
+      }
+      expect {
+        post passenger_trips_path(passenger.id), params: new_trip
```

Comment:
Consider whether the request body `params: new_trip` are actually used when creating a new trip.